src/hub.html

```diff
@@ -13,9 +13,9 @@
     <link href="https://fonts.googleapis.com/css?family=Zilla+Slab:300,300i,400,400i,700" rel="stylesheet">
 
     <% if(NODE_ENV === "production") { %>
```

Comment:
FYI: @gfodor has PR #217, which was originally to bump A-Frame to `0.8.2`: https://github.com/mozilla/hubs/pull/217/files

as I understand it, it's probably best to point to aframevr/aframe@3e7a4b3. likely, PR #217 can be closed?